django/db/migrations/loader.py

```diff
@@ -186,15 +186,20 @@ def build_graph(self):
         # This is just for dependency re-pointing when applying replacements,
         # so we ignore run_before here.
         reverse_dependencies = {}
-        for key, migration in normal.items():
+        reverse_dependencies_normal = {}
```

Comment:
I used this naming to be consistent with the existing code (lines 177-179) where 'normal' refers to non-replacing migrations. I'll add a comment to this to explain it better. Unless you have a suggestion for a better name?